presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestingConnectionProperties.java

```diff
@@ -13,12 +13,21 @@
  */
 package com.facebook.plugin.arrow;
 
-public class TestingConnectionProperties
+public final class TestingConnectionProperties
 {
-    public String database;
-    public String password;
+    public final String database;
```

Comment:
these can be private, with getters added for them

Reply:
done